Example/TABTestKit_ExampleUITests/BiometricLogin_HappyPathTests.swift

```diff
@@ -23,6 +23,13 @@ final class BiometricLogin_HappyPathTests: TABTestCase, SystemPreferencesContext
       When(I: complete(biometricLoginScreen))
       Then(I: doNotSee(biometricLoginScreen))
     }
+    
+    Scenario("Relaunching the app") {
+      Given(I: doNotSee(biometricLoginScreen))
+      When(I: relaunchTheApp)
+      Then(I: amBackToTABTestKitExampleApp)
```

Comment:
Would a function name like `goBackToTABTestKitExampleApp()` make more sense here? We infer we're back in the app in the line after this.